### PR TITLE
Create keli5.json

### PR DESCRIPTION
**Pull request by old-df38257622**

### CODE REVIEW

File: domains/keli5.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "keli5",
+  "repo": "https://github.com/keli5/keli5.github.io",
+  "owner": {
+    "username": "keli5",
+    "email": ""
```
Comment (phenax) [inline domains/keli5.json:6]: You should specify an email address so it is easier to get in touch with you. Or if you use any other social media platform, you can type your username for that in a seperate key as `"twitter": "whatever"` or a`"matrix": "whatever"`

File: domains/keli5.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  },
+  "record": {
+    "CNAME": "keli5.github.io"
```
Comment (phenax) [inline domains/keli5.json:9]: To force https on the website, have a look at https redirection in [the docs](https://github.com/is-a-dev/register/blob/main/API.md#record-required)

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/keli5.json (hunk 2 above)
Comment (keli5) [reply domains/keli5.json:9]: My site doesn't support HTTPS and won't for a while yet.

File: domains/keli5.json (hunk 1 above)
Comment (keli5) [reply domains/keli5.json]: Done, added twitter

**Review by phenax — APPROVED** [review]

**Comment by phenax** [comment]
The record has been published. To fix the 404, add your-domain.is-a.dev to your github pages repo in Settings > Github pages > Custom domain
